.github/workflows/python-publish.yml

```diff
@@ -26,7 +26,9 @@ jobs:
 
       - name: Build wheel
         env:
-          CIBW_SKIP: cp27-* pp27-* pp36-*
+          CIBW_BUILD: cp3?-*
+          CIBW_SKIP: cp35-* *-win32 *-win_amd64 *-manylinux_i686 *-manylinux_aarch64 *-manylinux_ppc64le *-manylinux_s390x
```

Comment:
Can you put a comment on this to simply say it builds and tests for CPython >=3.6 on x86_64 architecture?